easybuild/easyblocks/c/cp2k.py

```diff
@@ -342,28 +369,29 @@ def configure_common(self):
         cflags = os.getenv('CFLAGS')
         fflags = os.getenv('FFLAGS')
         fflags_lowopt = re.sub('-O[0-9]', '-O1', fflags)
+
         options = {
             'CC': os.getenv('MPICC'),
-            'CPP': '',
+            'AR': 'ar -r',
             'FC': '%s %s' % (os.getenv('MPIF90'), self.openmp),
             'LD': '%s %s' % (os.getenv('MPIF90'), self.openmp),
-            'AR': 'ar -r',
-            'CPPFLAGS': '',
-
             'FPIC': self.fpic,
+            'DFLAGS': ' -D__parallel -D__BLACS -D__SCALAPACK -D__FFTSG %s' % self.cfg['extradflags'],
+            'INCS': '',
+            'CFLAGS': '-O3 -fopenmp -ftree-vectorize -march=native -fno-math-errno -fopenmp -std=c11 $(FPIC) $(DEBUG) '
```

Comment:
This is not the way to do it, do not hardcode compiler flags like this.
-march=native only works for GCC for instance.
The old code was correct apart from the missing INCS and DFLAGS.